docs/SCHEMA.md

```diff
@@ -60,6 +61,7 @@ Valid values are:
 - `secret` (the same as text but UIs might decide to hide the user input)
 - `list(<X>)` (a comma separated list of values, where `<X>` defines any of the formats above). Example: `list(email)` for `support@twilio.com,open-source@twilio.com`. Use `text` for any type
 - `map(<X>,<Y>)` (a comma and semicolon separted list. often used for key value pairs. `<X>` and `<Y>` represent the format for the key and value). Example: `map(email,phone_number)` for: `help@twilio.com,+1222333444;support@twilio.com,+13334445555`
+- `file(<X>)` (the path to a file in format `<X>`, which currently only supports `json`). Example: `file(json)` for `config/test.json`
```

Comment:
Its a little nuanced but it should be the deployed path. Such that we have the `<deployed url>+path` to access the resource that is useable within the function files

Reply:
It should actually support absolute values in my opinion. Because the way you'd use it is something like:
```
fs.readFile(Runtime.getAssets()[context.ENV_VARIABLE].path)
```

For this the `context.ENV_VARIABLE` would have to be a path such as `/config/test.json`.